### ISSUE TITLE
Tree item has focus ring when navigating using mouse (Teams Theme)

### ISSUE THREAD

**Issue by miroslavstastny** [microsoft/fluent-ui-react#1620]
### Steps
Click on a tree item.

### Expected Result
No focus ring visible, Teams theme displays focus ring only when navigating using keyboard.

### Actual Result
![image](https://user-images.githubusercontent.com/9615899/61063678-a95e5e00-a400-11e9-8f6f-2bf92d4b983a.png)

### Version
0.34.1
